spaceship/lib/spaceship/portal/app_service.rb

```diff
@@ -30,85 +30,44 @@ def initialize(service_id, value)
         end
       end
 
-      class << self
-        def app_group
-          self::AppGroup
-        end
-
-        def apple_pay
-          self::ApplePay
-        end
-
-        def associated_domains
-          self::AssociatedDomains
-        end
-
-        def data_protection
-          self::DataProtection
-        end
-
-        def game_center
-          self::GameCenter
-        end
-
-        def health_kit
-          self::HealthKit
-        end
-
-        def home_kit
-          self::HomeKit
-        end
-
-        def wireless_accessory
-          self::WirelessAccessory
-        end
-
-        def icloud
-          self::Cloud
-        end
-
-        def cloud_kit
-          self::CloudKit
-        end
-
-        def in_app_purchase
-          self::InAppPurchase
-        end
-
-        def inter_app_audio
-          self::InterAppAudio
-        end
-
-        def passbook
-          self::Passbook
-        end
-
-        def push_notification
-          self::PushNotification
-        end
-
-        def siri_kit
-          self::SiriKit
-        end
-
-        def vpn_configuration
-          self::VPNConfiguration
-        end
-
-        def network_extension
-          self::NetworkExtension
-        end
-
-        def hotspot
-          self::Hotspot
-        end
-
-        def multipath
-          self::Multipath
-        end
-
-        def nfc_tag_reading
-          self::NFCTagReading
+      def self.new_service(id, values: { on: true, off: false })
```

Comment:
This is a bit of a larger refactor, for two reasons:

1. The require checker can't cope with the constants that are being defined at the end of the file being accessed here
2. It was kinda ugly, having to define the same methods over and over again, when we can have ruby easily do it for us, so we saved some 250 lines